spec/index.html

```diff
@@ -93,7 +93,7 @@ <h2>Introduction</h2>
     also known as a <a data-cite="RDF12-CONCEPTS#dfn-quad">quad</a>.
     These may be separated by white space (spaces <code>#x20</code> or tabs <code>#x9</code>).
     This sequence is terminated by a '<code>.</code>'
-    (optionaly proceded by white space and/or a comment),
+    (optionally proceded by white space and/or a comment),
```

Comment:
By `proceded`, did you mean `followed` or `preceded`? I'm guessing the latter...
```suggestion
    (optionally preceded by white space and/or a comment),
```

Reply:
Should be "followed".

```suggestion
    (optionally followed by white space and/or a comment),
```